codecov_auth/management/commands/tests/test_set_trial_status_values.py

```diff
@@ -1,4 +1,4 @@
-from datetime import datetime, timedelta
+from datetime import UTC, datetime, timedelta
```

Comment:
I don’t see this import being used.

Reply:
Removed in https://github.com/codecov/codecov-api/pull/757/commits/4eb66ead7fa29cd926d7d71a679994962a76507f, thanks!